scripts/flashing/firmware_utils.py

```diff
@@ -221,7 +221,15 @@ def run_tool(self,
                     capture_output=True)
             else:
                 result = self
-                self.err = subprocess.call(command)
+                self.error = subprocess.check_call(command)
+        except subprocess.CalledProcessError as exception:
+            self.err = exception.returncode
+            if capture_output:
+                self.log(fail_level, '--- stdout ---')
+                self.log(fail_level, exception.stdout)
+                self.log(fail_level, '--- stderr ---')
+                self.log(fail_level, exception.stderr)
+                self.log(fail_level, '---')
```

Comment:
nit: shouldn't be better to have only one single log entry?
```suggestion
                self.log(fail_level, '--- stdout ---\n'
                    f'{exception.stdout}\n--- stderr ---\n'
                    f'{exception.stderr}\n---')
```

Reply:
Lets follow up in a separate PR if needed. I would like to have this version in to catch the reason of the flakyness. In most cases these logs will not be used.

Reply:
Unresolved after merge. @kpschoedel could you followup in a separate PR or argue that it is not needed? 

there would also be the option of 'every stdout/stderr line should have its own log line' so that things align nicely in logs.

Reply:
Yes, I'll address this one way or another, when we catch the parttool error or separately.